pkg/ds/daemon_set_test.go

```diff
@@ -306,13 +307,14 @@ func TestSchedule(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Unable to delete daemon set: %v", err)
 	}
-	numNodes = waitForNodes(t, ds, 0, desiresErrCh, dsChangesErrCh)
-	Assert(t).AreEqual(numNodes, 0, "took too long to unschedule")
+	// Behaivour change: Deleting a daemon set will no longer unschedule its pods (for now)
```

Comment:
you spelled it wrong, and I'm not talking about the ou, I'm talking about the iv